project/build.properties

```diff
@@ -1 +1 @@
-sbt.version = 0.13.18
+sbt.version = 1.4.0
```

Comment:
Any idea what this changes? I was having issues getting my tooling to work due to mixed versions of Scala, so I upgraded SBT.

Reply:
I guess a newer SBT shouldn't hurt, as long as it works.